draft-thomson-gendispatch-no-expiry.md

```diff
@@ -96,8 +96,8 @@ NEW:
 
 > Internet-Drafts are draft documents that may be updated, replaced, or
   obsoleted by other documents at any time. It is inappropriate to
-  cite them other than as "work in
-  progress."
+  cite them other than as "work in progress."  Drafts that have not been
+  updated after many months may have been abandoned.
```

Comment:
I don't think that this is necessary to add to boilerplate.

Reply:
Sorry, I forgot this in the commit message.  My point is that old drafts MAY get stale.  That is now becoming a matter of author intent.

Reply:
I'm with Martin on not including it in the boilerplate. There are many things that may have happened after many months, and "been abandoned" is just one of them.

Reply:
```suggestion
  cite them other than as "work in progress."
```

Reply:
Ok, then my request is that you continue to include the text that indicates that it may be inappropriate to cite as reference material.  Otherwise, you are entirely defeating the purpose of an Internet-Draft.